### PR TITLE
Minor fixes for the rabbitmq-server.ocf OCF resource agent

### PR DESCRIPTION
**Pull request by vuntz**
These are two minor fixes from the openSUSE package.

### CODE REVIEW

**Comment by michaelklishin** [comment]
@binarin @dmitrymex OK with you?

**Comment by michaelklishin** [comment]
@vuntz thank you, I assume this can go into `3.6.x` as well as master?

**Comment by vuntz** [comment]
> @vuntz thank you, I assume this can go into 3.6.x as well as master?

Having this in 3.6.x would be great, indeed.

File: scripts/rabbitmq-server.ocf [hunk 1]
```diff
@@ -299,7 +299,7 @@ rabbit_stop() {
         return $OCF_SUCCESS
     fi
 
-    $RABBITMQ_CTL stop ${RABBITMQ_PID_FILE}
+    rabbitmqctl_action stop
```
Comment (binarin) [inline scripts/rabbitmq-server.ocf]: Is there a reason to omit `${RABBITMQ_PID_FILE}` there?

Comment (michaelklishin) [reply scripts/rabbitmq-server.ocf]: I agree, this looks like an oversight. `${RABBITMQ_PID_FILE}` is used with e.g. `rabbitmqctl wait` in other places, for example.

Comment (vuntz) [reply scripts/rabbitmq-server.ocf]: Thanks, fixed.

**Review by michaelklishin — APPROVED** [review]

**Comment by michaelklishin** [comment]
Will backport to `stable`.

**Pull request merged**